target/linux/ramips/image/mt76x8.mk

```diff
@@ -119,6 +119,16 @@ define Device/glinet_vixmini
 endef
 TARGET_DEVICES += glinet_vixmini
 
+define Device/hak5_wifi-pineapple-mk7
+  IMAGE_SIZE := 32448k
+  DEVICE_VENDOR := Hak5
+  DEVICE_MODEL := WiFi Pineapple
+  DEVICE_VARIANT := Mark 7
```

Comment:
@adschm I'm adding this comment again as I did it in the outdated section before so it might got lost.

Why do you think DEVICE_VARIANT should be used here? All other Pineapples are based on different platforms (AR9331, AR9344, IPQ40xx). And they are basically different models/products, sharing only the family name. IMHO, that would be misuse of this feature here.

Reply:
> All other Pineapples are based on different platforms (AR9331, AR9344, IPQ40xx). And they are basically different models/products, sharing only the family name.

I agree with this, but it is not impossible (although, very unlikely) that a revision of the hardware could come in the future. Do we want to keep the PR as it is in case that ever happens, or prefer to remove DEVICE_VARIANT now and re-add it if a future revision ever happens?

Reply:
In addition to what I wrote above: DEVICE_MODEL/DEVICE_VARIANT will only affect `make menuconfig` and the image selector if it's finally deployed. It can be changed at any time without any considerations about migration, and from that point everything will be updated. So, we cannot do much harm here.

The more important choice is the device/image name "hak5_wifi-pineapple-mk7", which can't be updated that easily.

Reply:
@adschm from what I can see currently in code, we use `DEVICE_VARIANT` only for revisions of the same device or some kind of (sub-)type of the image (SD card, flash, type of NAND, etc.). I really don't think `DEVICE_VARIANT` should be used here.

@Foxtrot I would drop the `DEVICE_VARIANT` here and just use the `DEVICE_MODEL`. **If** someday a new revision of this model (I mean the same or very similar hardware, announced as for example `v2` or `Mark 7+`) gets released, then the `DEVICE_VARIANT` would make sense.